react_main/src/css/nav.css

```diff
@@ -214,7 +223,16 @@
 }
 
 @media (max-width: 800px) {
+  input[class="hamburger-box"]:checked ~ .sub-nav {
+    display: block;
+  }
+
+  .hamburger {
```

Comment:
this class is not defined anywhere, should it be subnav box?

is the intended effect for a menu to pop open